packages/indexer/src/vcr/replay.ts

```diff
@@ -0,0 +1,11 @@
+import { VcrClient } from "./client";
+import type { VcrConfig } from "./config";
+import type { Indexer } from "@apibara/indexer";
+
+export async function replay<TFilter, TBlock, TRet>(
```

Comment:
Looks good for now. I think in the future we will want to call `run` with `VcrClient` but I'm not sure yet.